### DISCUSSION TITLE
Welcome to discussions Discussions!

### DISCUSSION THREAD

**Discussion by avidrucker** [knowledge-for-good/knowledgeforgood#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by avidrucker** [comment]
Hi @seekanddefine !

**Comment by seekanddefine** [comment]
Warm greetings,

I am a passionate access, diversity, and inclusion advocate; a multi-disciplinary creative; and a life-long self-directed learner with 15 years experience in higher education and technology accessibility privileged to call rural Minnesota home as I raise and school my daughter. Social justice propels my daily work to empower people to claim the truth of their intersectional stories and unique bodies in order to find whole body-mind healing and sustainable wellbeing on an individual and global scale. 

I have played with words since I can remember and found learning a sacred, safe haven for a girl who could barely hear until surgery transformed that at nine years old. Since an Honors and English Bachelor's degree and Masters work in Linguistics, I have engaged joyfully in human services teaching teachers, tutors, and students; directing and creating learning centers; providing accessibility accommodations; ensuring ADA and WCAG compliance; editing, grant-writing, and researching; and advocating for social justice. I am deeply committed to equal access for all students and continued support to persist to completion through data-driven and  compassionate, trauma-informed communication.

**Reply by seekanddefine** [reply]
Was there a better place to put this? Should this discussion always remain open or would it be closed once current members feel they are done with it? Should this be closed? It appears we are done with it for the time being. @avidrucker

**Reply by seekanddefine** [reply]
Discussions are always open! I was thinking this was an issue, my bad.

**Reply by avidrucker** [reply]
We may want to link to your awesome bio here, perhaps from the readme?
